db/system_keyspace.hh

```diff
@@ -128,6 +128,7 @@ class system_keyspace : public seastar::peering_sharded_service<system_keyspace>
     static schema_ptr large_cells();
     static schema_ptr scylla_local();
     future<> force_blocking_flush(sstring cfname);
+    future<> ensure_peers_table_fixup();
```

Comment:
Name is vague. I suggest to either use a more descriptive name (`remove_old_ips_from_peers_table`? but it doesn't cover the host ID part -- which I'm not a fan of though, comment above) or a comment (preferably both)

Reply:
renamed the functions, added a comment